src/enrollments/enrollments.dtos.ts

```diff
@@ -0,0 +1,26 @@
+import { IsNumber } from "class-validator";
+
+export class CreateEnrollmentDto {
+  @IsNumber()
+  studentId!: number;
+
+  @IsNumber()
+  classroomId!: number;
+}
+
+export class EnrollmentResponseDto {
+  @IsNumber()
+  id!: number;
+
+  @IsNumber()
+  studentId!: number;
+
+  @IsNumber()
+  classroomId!: number;
+
+  constructor(id: number, studentId: number, classroomId: number) {
+    this.id = id;
+    this.studentId = studentId;
+    this.classroomId = classroomId;
+  }
```

Comment:
Do you need all this to be sent via the response upon successful enrollment?

Reply:
I thought I need studentId and classroomId for sending notification via mail. But as I already implemented the mail logic I don't need this response, instead I will return true if the enrollment is successful